docs/_summary.md

```diff
@@ -116,6 +116,7 @@
     * [Stenography](feature_stenography.md)
     * [Thermal Printer](feature_thermal_printer.md)
     * [Velocikey](feature_velocikey.md)
+    * [Programmable Button](feature_programmable_button.md)
```

Comment:
This should be placed above Proton C Conversion.

Reply:
Actually, shouldn't fit better into "Software Features" or "Simple/Advanced Keycodes" instead of hardware features?

Reply:
Yes, probably. As long as it is in alphabetical order, wherever it lands.